UX/ui-service/MyServices/design.md

```diff
@@ -3,48 +3,62 @@
+### Service Details - Summary View
+* The properties section at the top of the summary view will be consistent for ALL services.
+![Image of Detail View for a Service.](img/Services-GenericDetailView.png)
+* Properties will consist of general information, including tags. Utilization properties will be excluded from this section and will show up in the dashboard view instead.
+* The next section of the summary view will vary depending on the type of service the user has selected. For legacy services there will be a resources section and for Ansible services there will be a Results section.
+
+#### Service Details - Summary View - Legacy Services
+* For legacy services, the next section under the properties will be the resources.
+![Image of Detail View for a Legacy Service.](img/Services-LegacyDetailView.png)
+* Resources will be divided into sections or headings (Compute, Network, and Storage). Each section can expand and collapse, with a compact list view of resources underneath.
+
+* Individual resources will be listed as links so users can drill down to a specific resource details view.
 ![Image of Detail View for a Resource.](img/Services-ResourceDetails.png)
+* **Research** Determine actions available for resources. Should these be listed within the standard button groups?
+* Relationships will be the next section after Resources. Child Services will be listed as links here and will bring the user to the Child Service Summary View.
 
-3. **Relationships:** Child Services will be listed as links and will bring the user to the Child Service Summary View.
+#### Service Details - Summary View - Ansible Services
+* For Ansible services, the next section under the properties will be the results and standard out.
+![Image of Detail View for an Ansible Service.](img/Services-AnsibleDetailView.png)
+* The Results section will have a basic set of information regarding the service, followed by a compact list view of plays.
+* The Standard Out section will include a log of the most recent information available from the time the page was loaded. If the play is still in progress, a status will appear on the right, along with a button to "Watch Live" details of the play. This live version will be displayed in a modal.
 
 #### Child Service - Summary View
 ![Image of Detail View for a Child Service.](img/Services-ChildDetailView.png)
 * The breadcrumbs will change to show My Services and the Parent Service as links.
 * The service details page will be in the same format as the Parent Service Detail page.
 
+### Service Details - Dashboard View
+* The dashboard view will display all utilization properties at the top of the page, followed by a utilization trend card below.
+* The trend card will include donut charts to display the CPU, Memory, and Storage data.
+![Image of Dashboard View for a Service.](img/Services-DashboardDetailView.png)
```

Comment:
@serenamarie125  Agreed, it would be ideal if we can remove the list of values and present them in the trend card instead. I also agree, we can remove the actions :)